src/script/page/MainContent/panels/preferences/DevicesPreferences/DevicesPreference.tsx

```diff
@@ -61,7 +61,11 @@ export const DevicesPreferences: React.FC<DevicesPreferencesProps> = ({
   const [localFingerprint, setLocalFingerprint] = useState('');
 
   const {devices} = useKoSubscribableChildren(selfUser, ['devices']);
-  const {getDeviceIdentity} = useUserIdentity(selfUser.qualifiedId, conversationState.selfMLSConversation()?.groupId);
+  const {getDeviceIdentity} = useUserIdentity(
+    selfUser.qualifiedId,
+    conversationState.selfMLSConversation()?.groupId,
+    true,
```

Comment:
DevicesPreference is a self client's device component, this is what `true` indicates here.